components/script/script_thread.rs

```diff
@@ -692,8 +694,11 @@ pub struct ScriptThread {
     /// Code is running as a consequence of a user interaction
     is_user_interacting: Cell<bool>,
 
-    /// Channel to WebGPU
+    /// Identity manager for WebGPU resources
     gpu_id_hub: Arc<Mutex<Identities>>,
+
+    /// Receiver to receive commands from optional WebGPU server.
+    webgpu_port: RefCell<Option<Receiver<WebGPUMsg>>>,
```

Comment:
Shouldn't this be `DomRefCell`?

Reply:
I am not sure about this. `DomRefCell` is used for DOM objects, I don't think this is one. But again I am not sure. I'll change it if you say so and it would be great if you could help me understand the reason :)

Reply:
I see. I hope that the compiler will tell us if we mess this up, so we can leave it like this (unless @jdm tells otherwise)